diagnostic.md

```diff
@@ -40,25 +43,26 @@ like to show all movies favorited by a profile on
 `http://localhost:3000/profiles/1`
 
 ```sh
-  # < Your Response Here >
+  It is a way of transfering data.
 ```
 
 ```rb
 class ProfileSerializer < ActiveModel::Serializer
+attributes :Profiles
 end
 ```
 
 1.  What would the command be to _scaffold_ out a **join table** for Favorites from
 the above `Movies` and `Profiles`.
 
 ```sh
-  # < Your Response Here >
+  bundle exec rails g scaffold Favorites
 ```
 
 1.  What is `Dependent: Destroy` and where/why would we use it?
 
 ```sh
-  # < Your Response Here >
+  All elements will be deleted as well as destroyed upon use, in a model file.
 ```
```

Comment:
When an instance of a class is destroyed in the model, it also deletes all associated objects.